external-dependencies/populate.sh

```diff
@@ -35,8 +35,10 @@ cp h5py/h5py-2.10.0-cp39-cp39-win_amd64.whl .
 # Copy the compiled windows scipy to external dependencies
 cp scipy/scipy-1.6.2-cp39-cp39-win_amd64.whl .
 
-# Download windows wheel (cp39, win, amd64) from https://pypi.org/simple/netcdf4/
+# Download windows wheels (cp39, win, amd64)
 wget https://files.pythonhosted.org/packages/92/2e/8a85d66cd4646a81027f3efbc17278beaa86fd351b5f1339a7a5b0d79c14/netCDF4-1.5.4-cp39-cp39-win_amd64.whl#sha256=223b84f8d2a148e889b1933944109bdecbefc097200ab42e8a66c967b1398e1b
+wget https://files.pythonhosted.org/packages/6f/4c/1cdac6e8036f68d0a546118c78b35855b6e7ed8a7010d23ac48f1984051d/pygeos-0.12.0-cp39-cp39-win_amd64.whl
+wget https://files.pythonhosted.org/packages/83/83/664c4e0c286d2dbf8294147493263e0991b37538e594b591504ddedf6881/threedigrid_builder-1.3.6-cp39-cp39-win_amd64.whl
```

Comment:
Aren't the linux versions of these wheels also necessary (for testing)?

@benvanbasten-ns : I don't know how Gijs handles the mac stuff? Should those wheels also be downloaded?

Reply:
@reinout We don't handle that. We haven't finished the Mac implementation. The plugin loads, but some particular tools still fail and require additional research.